src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -82,6 +85,8 @@
     private static final int MAX_BUF_SIZE = Integer.MAX_VALUE;
     // data size when buffer is divided up to aid in intrinsics
     private static final int TRIGGERLEN = 65536;  // 64k
```

Comment:
With this interleaved impl, is this TRIGGERLEN still needed? The implGCMCrypt(byte[] in, int inOfs, int inLen,
        byte[] ct, int ctOfs, byte[] out, int outOfs, GCTR gctr, GHASH ghash) method is intrinsified, would there be a difference in increasing the number of gctr/ghash calls inside an already intrinsified method?

Reply:
Yes, they are two different intrinsics.  The new intrinsic is supported by newer processors so there is no guarantee that implGCMCrypt will run the intrinsic.

Reply:
Hmm, ok. Thanks for the explanation.